src/boot/auth-header.ts

```
@@ -0,0 +1,142 @@
+        : (input as Request).url;
+
+  let cfg = init;
+
+  if (isApi(url)) {
+    // Merge headers: start with Request.headers (if input is a Request),
+    // then overlay init.headers (caller intent wins), then add Authorization if absent.
+    const merged = new Headers(
+      input instanceof Request ? (input as Request).headers : undefined
+    );
+    if (init?.headers) {
+      new Headers(init.headers).forEach((v, k) => merged.set(k, v));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)